x/concentrated-liquidity/types/params.go

```diff
@@ -0,0 +1,70 @@
+package types
+
+import (
+	fmt "fmt"
+	"strconv"
+
+	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
+)
+
+// Parameter store keys.
+var (
+	KeyAuthorizedTickSpacing = []byte("AuthorizedTickSpacing")
+
+	_ paramtypes.ParamSet = &Params{}
+)
+
+// ParamTable for concentrated-liquidity module.
+func ParamKeyTable() paramtypes.KeyTable {
+	return paramtypes.NewKeyTable().RegisterParamSet(&Params{})
+}
+
+func NewParams(authorizedTickSpacing []string) Params {
+	return Params{
+		AuthorizedTickSpacing: authorizedTickSpacing,
+	}
+}
+
+// DefaultParams returns default concentrated-liquidity module parameters.
+func DefaultParams() Params {
+	return Params{
+		AuthorizedTickSpacing: []string{"1", "10", "60", "200"},
+	}
+}
+
+// Validate params.
+func (p Params) Validate() error {
+	if err := validateTicks(p.AuthorizedTickSpacing); err != nil {
+		return err
+	}
+	return nil
+}
+
+// ParamSetPairs implements params.ParamSet.
+func (p *Params) ParamSetPairs() paramtypes.ParamSetPairs {
+	return paramtypes.ParamSetPairs{
+		paramtypes.NewParamSetPair(KeyAuthorizedTickSpacing, &p.AuthorizedTickSpacing, validateTicks),
+	}
+}
+
+// validateTicks validates that the given parameter is a slice of strings that can be converted to unsigned 64-bit integers.
+// If the parameter is not of the correct type or any of the strings cannot be converted, an error is returned.
+func validateTicks(i interface{}) error {
+	// Convert the given parameter to a slice of strings.
+	ticks, ok := i.([]string)
+	if !ok {
+		return fmt.Errorf("invalid parameter type: %T", i)
+	}
+
+	// Iterate over the slice of strings.
+	// For each string, attempt to convert it to an unsigned 64-bit integer.
+	// If the conversion fails, return an error.
+	for _, tick := range ticks {
+		_, err := strconv.ParseUint(tick, 10, 64)
```

Comment:
Can we make these as `uint64` in the first place? Do we have to store these as strings?